web/tests/scheduled_messages.test.js

```diff
@@ -0,0 +1,115 @@
+"use strict";
+
+const {strict: assert} = require("assert");
+
+const {zrequire} = require("./lib/namespace");
+const {run_test} = require("./lib/test");
+
+const {$t} = zrequire("./i18n");
+const scheduled_messages = zrequire("scheduled_messages");
+
+function get_expected_send_opts(expecteds) {
+    const modal_opts = {
+        send_later_tomorrow: {
+            tomorrow_nine_am: {
+                text: $t({defaultMessage: "Tomorrow at 9:00 AM"}),
+                time: "9:00 am",
+            },
+            tomorrow_four_pm: {
+                text: $t({defaultMessage: "Tomorrow at 4:00 PM"}),
+                time: "4:00 pm",
+            },
+        },
+        send_later_custom: {
+            text: $t({defaultMessage: "Custom"}),
+        },
+        possible_send_later_today: false,
+        possible_send_later_monday: false,
+    };
+    const optional_modal_opts = {
+        send_later_today: {
+            today_nine_am: {
+                text: $t({defaultMessage: "Today at 9:00 AM"}),
```

Comment:
We don't generally use `$t` in tests; this should be the actual string that the tests translate that into, which is "translated: Today at 9:00 AM". (This pattern lets us verify that things are tagged for translation in the tests without having the tests depend on the actual set of language fixtures).

Reply:
Good to know! Pushed a fix for this.